post_processing/mse_tables.jl

```diff
@@ -57,17 +57,17 @@ all_best_mse["sphere_held_suarez_rhoe_int_equilmoist"][(:c, :uₕ, :components,
 all_best_mse["sphere_held_suarez_rhoe_int_equilmoist"][(:c, :ρq_tot)] = 34854.686028320226
 all_best_mse["sphere_held_suarez_rhoe_int_equilmoist"][(:f, :w, :components, :data, 1)] = 29565.88835318117
 #
-all_best_mse["edmf_single_column"] = OrderedCollections.OrderedDict()
-all_best_mse["edmf_single_column"][(:c, :ρ)] = 9.879929344257052e-5
-all_best_mse["edmf_single_column"][(:c, :ρe_tot)] = 0.05010398752695285
-all_best_mse["edmf_single_column"][(:c, :uₕ, :components, :data, 1)] = 15.633092708261552
-all_best_mse["edmf_single_column"][(:c, :uₕ, :components, :data, 2)] = 81.25287663937327
-all_best_mse["edmf_single_column"][(:c, :ρq_tot)] = 0.3567000179447617
-all_best_mse["edmf_single_column"][(:c, :turbconv, :en, :ρatke)] = 99.56872201272839
-all_best_mse["edmf_single_column"][(:c, :turbconv, :up, 1, :ρarea)] = 7.884506143708434
-all_best_mse["edmf_single_column"][(:c, :turbconv, :up, 1, :ρaθ_liq_ice)] = 7.888499297457295
-all_best_mse["edmf_single_column"][(:c, :turbconv, :up, 1, :ρaq_tot)] = 7.866565945507953
-all_best_mse["edmf_single_column"][(:f, :turbconv, :up, 1, :ρaw)] = 2.8234965025701935
+all_best_mse["TurbulenceConvection"] = OrderedCollections.OrderedDict()
```

Comment:
The keys correspond to the `job_id` specified in the buildkite yml

Reply:
So, we'll need to add new dicts for the new jobs (if we opt in to regression tests).